src/app/services/export-excel/excel-export-sample-1/excel-export-sample-1.component.html

```diff
@@ -1,6 +1,6 @@
 <p class="grid__wrapper">
-  <igx-grid #igxGrid1 [data]="data" [allowFiltering]="true" [cellSelection]="'single'" [width]="'1200px'"
-    [hideGroupedColumns]="true" [height]="'700px'">
+  <igx-grid #igxGrid1 [data]="data" [groupingExpressions]='expr' [allowFiltering]="true" [cellSelection]="'single'"
+    [hideGroupedColumns]="true" [width]="'900px'" [height]="'700px'">
```

Comment:
I think it is best to set the width to 100%.
```suggestion
    [hideGroupedColumns]="true" [width]="'100%'" [height]="'700px'">
```